web/themes/new_weather_theme/templates/partials/wind-direction.html.twig

```diff
@@ -0,0 +1,9 @@
+{#
+The +180 here is to account for the fact that wind direction
+is reported as a FROM direction, but our arrow points in the
+TO direction. So we just need to spin it 'round.
+#}
+<svg role="img" aria-hidden="true" data-wind-direction class="width-2 height-2 margin-left-05 padding-2px" style="transform: rotate({{ windDirection.angle + 180 }}deg);">
```

Comment:
In general looking great but on the current conditions the icon looks a little too small. I imagine it's the combination of padding and the width on the icon:

Compare the table with current conditions:  
![wind icon on the table](https://github.com/weather-gov/weather.gov/assets/150970973/7daeab19-080e-47cc-8345-9f4c5e4a5717)

![wind icon on current](https://github.com/weather-gov/weather.gov/assets/150970973/535b147d-187e-4669-b050-8535c2d45d23)

Playing around with the dev tools I found that setting the class on the svg to `width-105 height-105 text-middle` (removing the padding and margin) got the icon pretty close to the right size and centered. But I imagine this would cause an issue in the table?

The other thing is that I could only get the icon to more closely be centered with the text if I added a `translate: -5% -12%;` to the `transform`. But those are some magic numbers. I am not sure why the rotated svg isn't quite centering with the text without that.

Reply:
On it!

Reply:
There was an old `.item` class on the arrow's parent in the current conditions. That was meant for the old icons that we're not using anymore (wind and humidity), and it was further reducing the size. I yoinked that out and wrapped the wind speed in a flex container with center alignment for the second problem.

Reply:
![image](https://github.com/weather-gov/weather.gov/assets/142943695/193cefef-3151-4d3b-a174-8a392a41a91d)

![image](https://github.com/weather-gov/weather.gov/assets/142943695/cc0a5c15-6e54-4d40-a45a-46de3d47a5e0)